### ISSUE TITLE
[BUG] - Input's Label is Announced Twice by VoiceOver

### ISSUE THREAD

**Issue by kimskovhusandersen** [nextui-org/nextui#3965]
### NextUI Version

2.4.8

### Describe the bug

The screenreader announces the label of the input twice

### Steps to Reproduce the Bug or Issue

Navigate to https://nextui.org/docs/components/input
Focus on the Input using VoiceOver.
Observe that the label is announced twice.

### Expected behavior

The screenreader should announce the label of the input only once to ensure clear and concise information is provided to the user.

### Screenshots or Videos

<img width="847" alt="Screenshot 2024-10-30 at 16 03 52" src="https://github.com/user-attachments/assets/6842b830-d889-4c66-ad73-fb929e9745b6">

### Operating System Version

macOS

### Browser

Chrome

**Comment by wingkwong** [comment]
Closing in favor of v3. If the issue is reproducible in v3, please create a new one with a minimal reproducible example.
